dev/Materials/Reveal/RevealBrush_rs2_themeresources.xaml

```diff
@@ -929,21 +929,14 @@
                     <Grid x:Name="RootGrid" Background="{TemplateBinding Background}">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Normal">
-                                    <Storyboard>
-                                        <PointerUpThemeAnimation Storyboard.TargetName="RootGrid" />
-                                    </Storyboard>
-                                </VisualState>
+                                <VisualState x:Name="Normal" />
```

Comment:
do the tilt animations need to go from RevealStyles themselves?

Reply:
Nope, my bad :)